apps/backend/src/applications/applications.entity.ts

```diff
@@ -0,0 +1,44 @@
+import {
+    IsArray,
+    IsBoolean,
+    IsPositive,
+    IsEnum,
+  } from 'class-validator';
+  import { Entity, Column } from 'typeorm';
+import { ApplicationStatus } from './types';
+  
+  @Entity()
+  export class Application {
+    @Column({ primary: true, generated: true })
+    @IsPositive()
+    applicationId: number;
+
+    @Column()
+    @IsPositive()
+    userId: number;
+
+    @Column()
+    @IsPositive()
+    featureId: number;
+  
+    @Column('varchar')
+    @IsBoolean()
+    safety?: boolean;
```

Comment:
I'm not super familiar with data in Nest, so a couple questions here:
- Are the `@IsBoolean()` (and related decorators) necessary? I don't see them in the scaffolding repo - was there some edge case you ran into while building this?
- How does defining the column as `varchar` and typing it as a boolean end up storing the record? Is it stored as text (like "true" or "false")?
- Does marking the field as optional with `?` do the same thing as adding the `IsOptional()` decorator?